rust-wasm-id-allocator/wasm-id-allocator/src/lib.rs

```diff
@@ -17,11 +17,15 @@ const MAX_SAFE_INTEGER: i64 = (2 as i64).pow(53) - 1;
 #[wasm_bindgen]
 impl IdCompressor {
     #[wasm_bindgen(constructor)]
-    pub fn new() -> IdCompressor {
-        IdCompressor {
-            compressor: IdCompressorCore::new(),
+    pub fn new(session_id_string: String) -> Option<IdCompressor> {
```

Comment:
wasm wrapper never makes its own session uuids, so always take one

Reply:
So there is no longer a public parameterless constructor for creating a net-new compressor?

Reply:
Correct. Always passed from TS.